lib/srv/alpnproxy/common/protocols.go

```diff
@@ -91,6 +94,7 @@ var SupportedProtocols = []Protocol{
 	ProtocolRedisDB,
 	ProtocolSQLServer,
 	ProtocolSnowflake,
+	ProtocolElasticsearch,
```

Comment:
```suggestion
	ProtocolElasticSearch,
```

Reply:
`Elasticsearch` is preferred: https://github.com/gravitational/teleport/pull/15768#discussion_r958277535.